src/mem/Stats.cc

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 1996-2022 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+#include "mem/Pool.h"
+#include "mem/Stats.h"
+
+size_t
+Mem::GlobalStats(PoolStats &stats)
+{
+    MemPools::GetInstance().flushMeters(); /* recreate TheMeter */
```

Comment:
This copied comment incorrectly (and unnecessary) documents what flushMeters() does.

```suggestion
    MemPools::GetInstance().flushMeters();
```

Reply:
FWIW, I do not think that was supposed to be documenting what flushMeters does, but what the GlobalStats() activity needed doing.